src/qrcode/QRTabBar.tsx

```diff
@@ -38,9 +50,19 @@ export default function QRTabBar({ state, descriptors, navigation, position, qrS
     outputColorRange: [colors.dark, colors.light],
   })
 
-  const onPressClose = () => {
-    navigation.getParent()?.goBack()
-  }
+  const { closeIconComponent, onPressClose } = useMemo(
+    () =>
+      closeIcon === CloseIcon.BackChevron
+        ? {
+            closeIconComponent: <BackChevron color={color} />,
+            onPressClose: () => navigate(Screens.WalletHome),
```

Comment:
I think I was having issues with navigation when I first added it, not sure why though because the `getParent` navigation works. I'll change it back that